modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/side_panel_content/index.js

```diff
@@ -18,9 +18,15 @@ import {
 	OPEN_MODAL_FROM_IFRAME,
 } from 'frontend-taglib-clay/data_set_display/utils/eventsDefinitions';
 
+function handleOpenModalFromSidePanel(payload) {
+	window.top.Liferay.fire(OPEN_MODAL_FROM_IFRAME, payload);
```

Comment:
Instead of `window.top`, we should use 
```js
const openerWindow = Liferay.Util.getOpener();

openerWindow.Liferay.fire(OPEN_MODAL_FROM_IFRAME, payload);
```
`getOpener` along with `getTop` [cover many edge cases](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L461-L527).

There are few other uses like this in the PR.

Reply:
Perfect! Thank you Marko - I didn't know these Utils :)

Reply:
I asked for a re-review :) I've used the Utils as you suggested but i preferred `getTop` since `getOpener` returns the wrong window in case of nested side panels:

![image](https://user-images.githubusercontent.com/44569796/118474230-1fbdfc80-b70b-11eb-9a5a-74d5689e5bf4.png)